src/Screens/PushInbox.tsx

```diff
@@ -255,7 +255,7 @@ const PushInbox = () => {
         renderItem={({ item, index }) => {
           return getInboxItem(item, index);
         }}
-        keyExtractor={item => item.pushInstanceId}
+        keyExtractor={item => item.pushInstanceId ?? ""}
```

Comment:
Wouldn't `keyExtractor={(item, index) => item.pushInstanceId ?? index.toString()}` be more sensible.